### PR TITLE
Add event

### PR DESCRIPTION
**Pull request by Lan2u**

### CODE REVIEW

File: fixed-seeding/seed-events.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
                 "day": 4,
                 "hour": 2,
                 "minute": 0
-            }
+            },
+            "venue": "StAge"
```
Comment (Vitineth) [inline fixed-seeding/seed-events.json:19]: Are we introducing a venue microservice and if so will this always be the name of the venue or will it become an identifier for the venue instead?

File: integration-tests/test_events.py [hunk 2]
```diff
@@ -12,6 +15,25 @@ def make_query(params):
...
+    return json.dumps(
+        {
+            'year': year,
+            'month': month,
+            'day': day,
+            'hour': hour,
+            'minute': minute
+        }
+    )
+
```
[the remark this answers is not in the document]
Comment (Lan2u) [reply integration-tests/test_events.py:45]: I'm switching to UTC epoch timestamps, this was a temp solution

File: fixed-seeding/seed-events.json (hunk 1 above)
Comment (Lan2u) [reply fixed-seeding/seed-events.json:19]: Idk, I reckon there are probably enough microservices for now to worry about / I think venue being handled by the same microservice isn't a problem currently.

An id to refer to a venue means:
ADV:  
 - No ambiguity to which venue is being referred to
 - Venues can change name easily
DisAdv:
 - We have to make calls to the venue database to check if the venue exists
 - The database becomes harder to debug as its less human readable - we need to basically implement SQL JOIN logic to get the venue name in returns (or return the ID and then have the front end do another query to match ID to venue name).

IMO the advantages outweigh the disadvantages and we should use a venue ID eventually.

Comment (Vitineth) [reply fixed-seeding/seed-events.json:19]: I think migrating to IDs eventually would be good, we can work out whether placing the work of resolving venues is better on the frontend or the backend later on as its not a priority right now. We can start with keeping venues in the same microservice but if things start to get out of hand or if we have a completely stable platform we can start considering migrating it out if we want

Comment (Lan2u) [reply fixed-seeding/seed-events.json:19]: Awesome, can you create an issue for this so we remember to come back to it later - also on the topic of issues I'll make an issue template

**Pull request merged**